v3/plugins/eclipse/che-theia/next/meta.yaml

```diff
@@ -67,3 +67,15 @@ spec:
          - exposedPort: 13132
          - exposedPort: 13133
      memoryLimit: "512M"
+  initContainers:
+  - name: remote-runtime-injector
+    image: aandrienko/che-theia-endpoint-runtime-binary:next
```

Comment:
How does versioning work for the endpoint binary image? Will we have to manually update to e.v. `:7.3.0` in the release process?

Reply:
Image should builds with all images in the che-theia project and after release we will have version :7.3.0 and latest will be updated too and should points to the :7.3.0.

Reply:
And yes we need manually update image version on release.

Reply:
We should make sure this is documented in the release process for 7.3.0.